gems/quilt_rails/test/generators/quilt/demo_app/demo_app_generator_test.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+require 'rails/generators'
+require 'generators/quilt/demo_app/demo_app_generator'
+
+class QuiltDemoAppGeneratorTest < Rails::Generators::TestCase
+  tests Quilt::DemoAppGenerator
+  destination File.expand_path("./tmp", File.dirname(__FILE__))
+
+  setup do
+    prepare_destination
+  end
+
+  test "creates the main React app" do
```

Comment:
really not sure why all the test are failing 😕

Reply:
The dummy app isn't present in the generator's test directory by default. It looks like this project sources files from two locations as well, which is redundant. Try changing `test/support/generator_test_helper.rb` to something like this:
```
module GeneratorTestHelpers
  DUMMY_PATH = File.expand_path("../dummy", __dir__)

  def copy_from_dummy(*path)
    dummy_path = File.join(DUMMY_PATH, *path)
    destination_path = File.join(destination_root, *path)

    FileUtils.mkdir_p(File.dirname(destination_path))
    FileUtils.cp(dummy_path, destination_path)
  end
end
```

And then calling it in the test cases like:
```
setup do
    prepare_destination
    copy_from_dummy("config", "application.rb")
  end
```
You'll need to update `QuiltReactAppGeneratorTest` to use `copy_from_dummy("config", "routes.rb")` with this approach too.